src/Microsoft.AspNetCore.Http.Abstractions/Microsoft.AspNetCore.Http.Abstractions.csproj

```diff
@@ -20,6 +20,7 @@ Microsoft.AspNetCore.Http.HttpResponse</Description>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.ActivatorUtilities.Sources" PrivateAssets="All" Version="$(MicrosoftExtensionsActivatorUtilitiesSourcesPackageVersion)" />
+    <PackageReference Include="Microsoft.Extensions.PropertyHelper.Sources" Version="$(MicrosoftExtensionsPropertyHelperSourcesPackageVersion)" PrivateAssets="All" />
```

Comment:
Why?

Reply:
Ah found it.

Reply:
RouteValueDictionary does some reflection when taking an `object`.